simd/src/Kokkos_SIMD_AVX2.hpp

```diff
@@ -958,11 +958,15 @@ class const_where_expression<simd_mask<double, simd_abi::avx2_fixed_size<4>>,
     }
   }
 
-  friend constexpr auto const& Impl::mask<double, abi_type>(
-      const_where_expression<mask_type, value_type> const& x);
+  [[nodiscard]] KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION value_type const&
```

Comment:
What happened to the constexpr?
Why did you add [[nodiscard]]?
(Same questions on all occurences)

Reply:
I don't like the `[[nodiscard]]` so much but that is fine.